torchquad/utils/enable_cuda.py

```diff
@@ -2,6 +2,7 @@
 import os
 
 import logging
+from utils.set_precision import set_precision
```

Comment:
Fails with 
![grafik](https://user-images.githubusercontent.com/2656685/117481925-f3b0b780-af63-11eb-9fb8-5812c77ba633.png)

Reply:
when I try to do
`import torchquad`